readthedocs.yml

```
@@ -6,7 +6,9 @@ formats:
 build:
   os: ubuntu-20.04
   tools:
-    python: "3.9"
+    python: "3.10"
+  apt_packages:
+    - pandoc
```

```suggestion
    python: "3.10"
```

it's already included